app/controllers/api/v0/articles_controller.rb

```diff
@@ -104,6 +104,23 @@ def me
           .decorate
       end
 
+      def reading_list
+        per_page = (params[:per_page] || 30).to_i
+        num = [per_page, 1000].min
```

Comment:
I think the max per_page should be a bit lower here given the complexity of the SQL statement below. How about 100? It also might be a good idea to pull this out and set it as a constant.
```
MAX_READING_LIST_PAGE_SIZE=100 # or something like that
```